modules/bitgo/src/v2/internal/tssUtils.ts

```diff
@@ -25,6 +25,7 @@ import { BitGo } from '../../bitgo';
 import { MpcUtils } from './mpcUtils';
 import { Memo, Wallet } from '..';
 import { RequestTracer } from '../internal/util';
+import { Uint8ArrayTobase58 } from '@bitgo/account-lib/dist/src/coin/sol/utils';
```

Comment:
let's avoid using Sol's util for base58 conversion.

you can do `import bs58 from 'bs58'`

Reply:
Fixed